include/NovelRT/Maths/Utilities.h

```diff
@@ -43,9 +56,14 @@ namespace NovelRT::Maths
             return (address + (alignment - 1ULL)) & ~(alignment - 1ULL);
         }
 
+        /**
+         * @brief Popcount is used to count the number of ones in a given word
+         * @param value A 32 bits unsigned integer to evaluate by counting the ones in bits of a given word
+         * @returns Number of ones in a given word value
+         */
```

Comment:
Just punctuation and stuff here!
```suggestion
        /**
         * @brief PopCount is used to count the number of active bits in a given word.
         * @param value A uint32_t to evaluate by counting the ones in bits of a given word.
         * @returns A count of the number of active bits in a given word value.
         */
```